### DISCUSSION TITLE
Can the version of OpenGFX affect gameplay beyond graphics?

### DISCUSSION THREAD

**Discussion by michalc** [OpenTTD/OpenTTD#12749] [Q&A] [answered]
I've been semi-assuming something, but want to check - can the version of OpenGFX affect gameplay beyond graphics? From what I sort of know: OpenGFX is a (collection of?) NewGRF file(s), which can affect gameplay beyond graphics - but does it / can it in future?

(Context: for https://github.com/michalc/OpenTTDLab I want to try to be as exhaustive as possible in terms of what can affect a game's behaviour)

**Comment by glx22** [comment, marked as answer]
OpenGFX is a basest, and basesets can't affect gameplay else network games would be impossible.

**Reply by michalc** [reply]
Thanks!

**Comment by andythenorth** [comment]
Graphics basesets (including OpenGFX) only affect sprites.

"OpenGFX+" however is a collection of standalone grfs that do have gameplay effects.  There are also other grfs on bananas that use "opengfx" in their names that may affect gameplay.

**Reply by michalc** [reply]
> There are also other grfs on bananas that use "opengfx" in their names that may affect gameplay.

Ah good to know - thanks!

**Comment by planetmaker** [comment]
Basesets by definition must not affect gameplay state - as otherwise different base sets could not play together.

However you have to take the broad approach to "graphics": the graphics influence how the game looks, including the UI and UI size. More so as the base sets can also provide the fonts, this may very well make the game look quite differently, including having different windows look very differently (similar to comparing the look in different languages). Thus don't try to compare screenshots or window sizes or such.

**Reply by michalc** [reply]
> Thus don't try to compare screenshots or window sizes or such.

Understood - thanks!
